src/main/java/com/github/fishio/PlayingField.java

```diff
@@ -167,6 +168,14 @@ public void addEntities() {
 	 * @return all the players in this field.
 	 */
 	public abstract ArrayList<PlayerFish> getPlayers();
+	
+	/**
+	 * @return
+	 * 		All the entities in this field.
+	 */
+	public List<Entity> getEntities() {
+		return new ArrayList<Entity>(entities);
```

Comment:
I disagree. It's usually good practice to make getMethods of Collections return a copy of that collection, so as not to expose the private field. And copy methods all work the same way, copying a ConcurrentLinkedQueue into another ConcurrentLinkedQueue should be as fast as copying a ConcurrentLinkedQueue into an ArrayList.